urdf/include/urdf/model.h

```diff
@@ -44,6 +44,7 @@
 #include <tinyxml.h>
 #include <boost/shared_ptr.hpp>
 #include <boost/weak_ptr.hpp>
+#include <ros/ros.h>
```

Comment:
`build_export_depend` is unsupported in `package.xml` format 1, which is what urdf is using. I believe the format 1 build_depend did that behavior by default.